content/md/imports-exports/exports.md

```diff
@@ -84,8 +84,7 @@ In the `Global settings` tab, properties depend on the export job. For example,
 | Delimiter     | Defines the character to delimit the fields in the CSV file _(not available in Serenity offer)_ |
 | Enclosure   | Defines the character for the field enclosure in the CSV file _(not available in Serenity offer)_ |
 | Number of lines per file | Defines the limit number of lines per file for XSLX export |
-| With header             | Defines if the first line of the exported file contains the columns names |
```

Comment:
`With header` is removed on purpose or is it a mistake?

Reply:
probably a mistake, fixed